### PR TITLE
expose controlplane in-cluster kubeconfig

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: pkg/certificate/certificate.go [hunk 1]
```diff
@@ -264,18 +264,37 @@ func InitKubeconfig(
 		return err
 	}
 
-	// expose the controlplane kubeconfig in a secret
+	// expose controlplane in-cluster kubeconfig in a secret
+	if err := util.KubeconfigWroteToSecret(
```
Comment (clyang82) [inline pkg/certificate/certificate.go:268]: who will use the in-cluster kubeconfig? is it for hosted mode case?

Comment (skeeey) [reply pkg/certificate/certificate.go:268]: yes

Comment (clyang82) [reply pkg/certificate/certificate.go:268]: Got it. you need rebase this PR.

Comment (skeeey) [reply pkg/certificate/certificate.go:268]: done

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**